Added non-WW non-multiple choice exercises, line 26

iankitnicolas opened the pull request:
The exercises currently on ORCCA in section 11.1 starting at 81 all use linear functions. I'm not sure if you wanted to have other types of functions on here like quadratic or with radicals, however since they do appear before 11.1 I assume it was okay to use for the sake of problem variety.

Let me know if I should rewrite these with linear functions if there are plans to reorder this section before quadratics or radicals.

iankitnicolas committed:
Added non-WW non-multiple choice exercises, line 26

Alex-Jordan commented:
These are great. Could you add one more? We usually want an even number of each kind of exercise, so that wherever we repeat an exercise, we can always say that the odd numbered one is there first, and the even numbered one that follows it is the copy. [These problems don't have to have duplicates, it's just that if problems followed this batch that had duplicates, we'd want that parity.]

Here are other things I wouldn't even mention except now you might be editing back in there anyway.

* `The function <m>M(t)=0.75t+100</m> `. I think we have tried to avoid grammar like this with function notation. Meaning "M(t)=0.75t+100" is not a function; it's "M" that is the function, and "M(t)=0.75t+100" is a formula for that function. So here you might do `The function <m>M</m> defined by <m>M(t)=0.75t+100</m> ` or something else like that. Just whatever avoids "The function <formula/> ..."
* `C(m)=\frac{1}{5}\sqrt{25m}` caught my eye, because why not simplify that to `C(m)=\sqrt{m}`? It could be like `C(m)=\frac{1}{5}\sqrt{30m}`, and then you still get integer output from using `m=120` or `m=270`.

iankitnicolas committed:
Fixed typo and added exercise line 26

iankitnicolas commented:
Great thanks for the feedback!

Here is a summary of what I added:

- The `function` part was a typo, I actually meant `formula`. But yes I understand the important grammatical difference.
- I simplified the radical expression to `C(m)=\frac{1}{2}\sqrt{m}` and changed some of the numbers accordingly.
- I added another exercise that uses a linear function with a negative slope, since I didn't have one previously.

Alex-Jordan commented:
These are great, thank you. Merging now.
